src/Castle.Core.Tests/TestDictionaryValidator.cs

```diff
@@ -47,11 +47,11 @@ public bool IsValid(IDictionaryAdapter dictionaryAdapter)
 		public string Validate(IDictionaryAdapter dictionaryAdapter)
 		{
 			List<String> errors = new List<string>();
-			var globalRules = AttributesUtil.GetTypeAttributes<IValidationRule>(dictionaryAdapter.Meta.Type);
+			var globalRules = AttributesUtil.GetTypeAttributes<ValidationRuleAttribute>(dictionaryAdapter.Meta.Type);

```

Comment:
Why did these unit tests need to change? I know I made changes to AttributesUtil. I just want to make sure we haven't broken functionality.